javascripts/related-articles.js

```diff
@@ -0,0 +1,55 @@
+const CERTAINTY_THRESHOLD = 0.92
```

Comment:
In addition to the `certainty` threshold we provide when querying for each articles nearest neighbors  in order to _build_ the  `related-articles.json` manifest — we also provide a second threshold here for post-filtering the items in the manifest before inserting them into the DOM. 

This isn't strictly necessary, but provided a convenient way to "tune" the results without having to rebuild the manifest each time I tweaked that `certainty` cutoff.